src/types/vendors.ts

```diff
@@ -53,3 +53,101 @@ export type UpdateVendorPayload = {
   tags?: string[]
   comment?: string
 }
+
+export type SkillsData = {
+  id: string
+  name: string
+}
+
+export type GetSkillsPayload = {
+  data?: SkillsData[]
+}
+
+export interface Prices {
```

Comment:
**[blocker]** (potentially)

Looks essentially like a duplicate of the "Price" type we have under "types/price", although I see some small differences. But pretty sure these should be the same type